controllers/food_controller.js

```diff
@@ -20,4 +20,14 @@ module.exports = class FoodController {
         res.status(404).send(error)
       })
   }
+
+  static remove(req, res) {
```

Comment:
If we're following REST, should this be destroy?

Reply:
changed